README.md

```diff
@@ -2,92 +2,707 @@
+  - [Downloading Objects](#downloading-objects)
+- [Advanced Configuration](#advanced-configuration)
+  - [Specifying Endpoints](#specifying-endpoints)
+  - [Enabling Direct Reads](#enabling-direct-reads)
+  - [Specifying Preferred Caches](#specifying-preferred-caches)
+- [Authorization](#authorization)
+  - [1. Providing a Token via Headers](#1-providing-a-token-via-headers)
+  - [2. Environment Variables](#2-environment-variables)
+  - [3. Default Token Location](#3-default-token-location)
+  - [4. HTCondor Token Discovery](#4-htcondor-token-discovery)
+  - [Token File Formats](#token-file-formats)
+  - [Automatic Token Discovery](#automatic-token-discovery)
+  - [Token Scopes](#token-scopes)
+  - [Token Validation](#token-validation)
+- [Integration with Data Science Libraries](#integration-with-data-science-libraries)
+  - [Using with xarray and Zarr](#using-with-xarray-and-zarr)
+  - [Using with PyTorch](#using-with-pytorch)
+  - [Using with Pandas](#using-with-pandas)
+- [Getting an FSMap](#getting-an-fsmap)
+- [Monitoring and Debugging](#monitoring-and-debugging)
+  - [Access Statistics](#access-statistics)
+  - [Enabling Debug Logging](#enabling-debug-logging)
+- [API Reference](#api-reference)
+  - [PelicanFileSystem](#pelicanfilesystem)
+  - [OSDFFileSystem](#osdffilesystem)
+  - [PelicanMap](#pelicanmap)
+- [Examples](#examples)
+- [Troubleshooting](#troubleshooting)
+- [Contributing](#contributing)
+- [License](#license)
+- [Citation](#citation)
+- [Support](#support)
+
 ## Overview
 
-PelicanFS is a file system interface (fsspec) for the Pelican Platform.  For more information about pelican, see our [main website](https://pelicanplatform.org) or [Github page](https://github.com/PelicanPlatform/pelican). For more information about fsspec, visit the [filesystem-spec](https://filesystem-spec.readthedocs.io/en/latest/index.html) page.
+PelicanFS is a file system interface (fsspec) for the Pelican Platform. It provides a Python interface to interact with Pelican federations, allowing you to read, write, and manage objects across distributed object storage systems.
+
+For more information about Pelican, see our [main website](https://pelicanplatform.org) or [GitHub page](https://github.com/PelicanPlatform/pelican). For more information about fsspec, visit the [filesystem-spec](https://filesystem-spec.readthedocs.io/en/latest/index.html) page.
+
+For comprehensive tutorials and real-world examples using PelicanFS with geoscience datasets, see the [Project Pythia OSDF Cookbook](https://projectpythia.org/osdf-cookbook/).
+
+**Note on Terminology:**
+- In URL terminology, `pelican://` and `osdf://` are properly called **schemes**. While fsspec refers to these as "protocols," we use the term "scheme" throughout this documentation for technical accuracy.
+- Pelican is an **object storage** system. Remote items are called **objects** (analogous to files) and **collections** (analogous to directories), not files and directories.
```

Comment:
It felt strange to refer to objects/collections as analogous to files/directories without explaining how/why they're different. Maybe you can provide a bit more of the nuance here, or if you want to keep this section short, link to another later section that goes into more detail about the differences?